corehq/apps/locations/tasks.py

```diff
@@ -99,7 +99,7 @@ def update_users_at_locations(domain, location_ids, supply_point_ids, ancestor_i
     Update location fixtures for users given locations
     """
     from corehq.apps.users.models import CouchUser, update_fixture_status_for_users
-    from corehq.apps.locations.dbaccessors import mobile_user_ids_at_locations
+    from corehq.apps.locations.dbaccessors import user_ids_at_locations
```

Comment:
At the moment this will return the same result as `mobile_user_ids_at_locations`. After the data migration this will also include the actual web users.

Reply:
Actually, if the mapping update is made and new web users are created, this `user_ids_at_locations` function will also include those web users, which is OK.